tests/tensor_search/test_validation.py

```diff
@@ -764,36 +738,16 @@ def test_validate_mappings_invalid(self):
                         "nontext": True  # non-number
                     },
                 }
-            },
-            { # needs more nesting
-                "type": "multimodal_combination",
-                "weights": {
-                    "some_text": 0.5
-                }
-            },
-            {
-                "my_combination_field": { # this dict is OK
-                    "type": "multimodal_combination",
-                    "weights": {
-                        "some_text": 0.5
-                    }
-                },
-                "other_field": {
```

Comment:
Because we have refactored into a MappingObject, and then `mappings: Optional[Dict[str, MappingObject]]`, the first test no longer makes sense (in essence it not longer gets validated in MappingObject, it gets validated in AddDocsParams

Reply:
The second case is similar. In essence, we are testing two separate cases the first (my_combination_field) passes, and the second (other_field) fails but is already a test case.